packages/manager/modules/vps/src/veeam/modal/schedule-backup/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,8 @@
+{
+  "vps_configuration_veeam_schedule_backup_title": "Configurer l'heure du Backup automatisé",
+  "vps_configuration_veeam_schedule_backup_description": "Effectuer le Backup automatisé tous les jours à (Fuseau horaire UTC) : ",
```

Comment:
Maybe we should have the parameter in the translation in case it has to be located elsewhere in a different language

Reply:
Thanks for the observation.